src/main/java/com/o19s/es/explore/PostingsExplorerQuery.java

```diff
@@ -172,5 +178,16 @@ public DocIdSetIterator iterator() {
         public float score() throws IOException {
             return this.postingsEnum.freq();
         }
+
+        /**
+         * Return the maximum score that documents between the last {@code target}
+         * that this iterator was {@link #advanceShallow(int) shallow-advanced} to
+         * included and {@code upTo} included.
+         */
+        @Override
+        public float getMaxScore(int upTo) throws IOException {
+            //TODO?
```

Comment:
same here, this query is meant for pure scoring a subset of docs not for extracting a top-N over millions of docs. I'm fine removing the TODO.